ui/component/app/view.jsx

```diff
@@ -280,7 +297,24 @@ function App(props: Props) {
       )}
       {/* @endif */}
       {/* @if TARGET='web' */}
-      {showAnalyticsNag && !noNagOnPage && (
+      {currentlyDegradedPerformance && (
+        <Nag
+          type="error"
+          message={
+            <I18nMessage
+              tokens={{
+                more_information: <Button button="link" label={__('more')} href="https://status.lbry.com/" />,
+              }}
+            >
+              lbry.tv is currently experiencing issues. Try refreshing to fix it.
```

Comment:
Not really a fan of this copy

Reply:
lbry.tv performance may be degraded. You can try to use it, or wait 5 minutes and refresh. Please no crush us.